packages/addons/sveltekit-adapter/index.ts

```diff
@@ -0,0 +1,102 @@
+import { defineAddon, defineAddonOptions } from '@sveltejs/cli-core';
+import { exports, functions, imports, object, type AstTypes } from '@sveltejs/cli-core/js';
+import { parseJson, parseScript } from '@sveltejs/cli-core/parsers';
+
+type Adapter = {
+	id: string;
+	package: string;
+	version: string;
+};
+
+const adapters: Adapter[] = [
+	{ id: 'node', package: '@sveltejs/adapter-node', version: '^5.2.9' },
+	{ id: 'static', package: '@sveltejs/adapter-static', version: '^3.0.6' },
+	{ id: 'vercel', package: '@sveltejs/adapter-vercel', version: '^5.5.0' },
+	{ id: 'cloudflare-pages', package: '@sveltejs/adapter-cloudflare', version: '^4.8.0' },
+	{ id: 'cloudflare-workers', package: '@sveltejs/adapter-cloudflare-workers', version: '^2.6.0' },
+	{ id: 'netlify', package: '@sveltejs/adapter-netlify', version: '^4.4.0' }
+];
+
+const options = defineAddonOptions({
+	adapter: {
+		type: 'select',
+		question: 'Which SvelteKit adapter would you like to use?',
+		options: adapters.map((p) => ({ value: p.id, label: p.id, hint: p.package })),
+		default: 'node'
+	}
+});
+
+export default defineAddon({
+	id: 'sveltekit-adapter',
+	// TODO: description?
+	shortDescription: '',
+	homepage: 'https://svelte.dev/docs/kit/adapters',
+	options,
+	setup: ({ kit, unsupported }) => {
+		if (!kit) unsupported('Requires SvelteKit');
+	},
+	run: ({ sv, options }) => {
+		const adapter = adapters.find((a) => a.id === options.adapter)!;
+
+		// TODO: should we do this?
+		// removes previously installed adapters
+		sv.file('package.json', (content) => {
+			const { data, generateCode } = parseJson(content);
+			const devDeps = data['devDependencies'];
+
+			for (const pkg of Object.keys(devDeps)) {
+				if (pkg.startsWith('@sveltejs/adapter-')) {
+					delete devDeps[pkg];
+				}
+			}
+
+			return generateCode();
+		});
```

Comment:
This portion removes any previously installed adapters. Is this a good/bad idea? Thoughts?

Reply:
It's going to be the right thing to do most of the time, so it's probably fine. It'd be annoying in the case where you don't want it, but I think it'd be pretty rare that you're deploying to multiple places and in that case the add-on isn't going to be able to help you much anyway as you're going to have to do something rather custom